addons/debugger/addon.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Logs",
+  "description": "Adds blocks to log data to a log in Scratch.",
+  "info": [
+    {
+      "type": "notice",
+      "text": "Protip: remove logs once you're done debugging",
+      "id": "logprotip"
+    },
+    {
+      "type": "notice",
+      "text": "When creating a log format, you can use the ${sprite}, ${content} (what the log says) and ${type} (log, warn, or error) variables. Each log is on it's own line. ",
+      "id": "logformatmsg"
+    }
+  ],
+  "credits": [
+    {
+      "name": "GrahamSH"
+    },
+    {
+      "name": "retronbv"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Export format",
+      "id": "exportFormat",
+      "type": "string",
+      "default": "${sprite}: ${content} (${type})"
```

Comment:
i think this is too much. why would you need to change the log output format lol

Reply:
scratch addons is all about customization. more options like this = good.

Reply:
@jeffalo Depends, see #1262. More options can also cause problems.

Reply:
i think the customization provided by a custom log format is useful